packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -144,6 +144,13 @@ export const loadCallPage = async (pages: Page[]): Promise<void> => {
   }
 };
 
+/**
+ * Click off the Composite page
+ */
+export const clickOffPage = async (page: Page): Promise<void> => {
```

Comment:
nit: consider `clickOutsideOfPage` -- because we have on/off for camera, mic etx. `off` feels like it is turning something off 😅